src/PageTypes/DatedUpdateHolderController.php

```diff
@@ -161,17 +162,26 @@ public function parseParams($produceErrorMessages = true)
         if (isset($tag)) {
             $tag = (int)$tag;
         }
-        if (isset($from)) {
-            $from = urldecode($from);
-            $parser = DBDatetime::create();
-            $parser->setValue($from);
-            $from = $parser->Format('y-MM-dd');
-        }
-        if (isset($to)) {
-            $to = urldecode($to);
-            $parser = DBDatetime::create();
-            $parser->setValue($to);
-            $to = $parser->Format('y-MM-dd');
+        try {
+            if (isset($from)) {
+                $from = urldecode($from);
+                $parser = DBDatetime::create();
+                $parser->setValue($from);
+                $from = $parser->Format('y-MM-dd');
+            }
+            if (isset($to)) {
+                $to = urldecode($to);
+                $parser = DBDatetime::create();
+                $parser->setValue($to);
+                $to = $parser->Format('y-MM-dd');
+            }
+        } catch (InvalidArgumentException $e) {
+            if ($produceErrorMessages) {
+                $this->getRequest()->getSession()->set(self::TEMP_FORM_MESSAGE, _t(
+                    __CLASS__ . '.InvalidDateFormat',
+                    'Dates must be in "y-MM-dd" format.'
+                ));
+            }
```

Comment:
Is it safe to continue processing if the exception is thrown, since the `$to` and/or `$from` variables may not be in a valid state?

Reply:
In my local testing, the "Filter has been applied with the dates reversed." error gets trips some time.